packages/pass-style/test/errors.test.js

```diff
@@ -48,17 +51,17 @@ test('toPassableError, toThrowable', t => {
   // Since then, we changed `makeError` to make reasonable effort
   // to return a passable error by default. But also added the
   // `sanitize: false` option to suppress that.
-  t.false(Object.isFrozen(e));
+  t.false(Object.isFrozen(e) && !harden.isFake);
```

Comment:
I almost missed that this does not demorganize into the not of the normal idiom. Why not?

Given the non-equivalence, I don't know that it is worth normalizing this one.